### DISCUSSION TITLE
登录节点失败请检查网络连接EOF

### DISCUSSION THREAD

**Discussion by xswlhhh12345** [natfrp/wiki#219] [使用问题]
急急急,刚开始还可以用樱花,用了一段时间按就出现这种情况了,有无大佬帮我看看是什么问题?

1. frpc version: 0.42.0-sakura-5 (built: 2022-11-20 17:19)
 登录节点失败, 请检查网络连接: EOF
.......
frpc 已退出
隧道启动失败, 将在 10 秒后重试

**Comment by Synchronized1** [comment]
也遇到这个问题,需要在frpc的配置文件中添加
[common]
tls_enable=true
关键不知道在哪里修改可以生效 😄
